lib/src/components/Switch/Switch.tsx

```diff
@@ -2,14 +2,18 @@ import { slider, switchInputStyle, switchLayout } from "./switch.css";
 import { ISwitchProps } from "./switch.types";
 import "./switch-global-styles.css";
 
+type SwitchProps = ISwitchProps & JSX.IntrinsicElements["input"];
+
 export const Switch = ({
   status,
   onChange,
   isDisabled = false,
-}: ISwitchProps) => {
+  ...nativeProps
+}: SwitchProps) => {
   return (
     <label className={switchLayout}>
       <input
+        {...nativeProps}
```

Comment:
Also destructure className and append it to the input. Otherwise the classname will be overwritten

eg:
```
{
isDisabled=false,
className="",
...nativeProps
}
<input className={`${switchInputStyle} {className}`} {...nativeProps}
```